src/views/network/view_network.cpp

```diff
@@ -148,6 +148,14 @@ namespace big
 				ImGui::InputInt("##playercount", &g.session.player_magnet_count);
 			}
 
+			ImGui::Checkbox("Trust friends", &g.session.trust_friends);
+			if (ImGui::IsItemHovered())
+				ImGui::SetTooltip("Friends won't be flagged as modders or taken actions by reactions");
+			
+			ImGui::Checkbox("Trust this session", &g.session.trust_session);
+			if (ImGui::IsItemHovered())
+				ImGui::SetTooltip("Players in this session won't be flagged as modders or taken actions by reactions");
```

Comment:
Should be converted to `bool_command`, can be declared within this file.

For example like the below commands:
https://github.com/YimMenu/YimMenu/blob/da028f03b3120a50de5e93bf1d4bc9a07fe9d3f8/src/views/view_vehicle_control.cpp#L317-L320